### PR TITLE
HYDRA-1303 : fix error with aiAreaLight at hydra startup.

### PR DESCRIPTION
**Pull request by lanierd-adsk**

### CODE REVIEW

File: lib/mayaHydra/hydraExtensions/plugInfo.json [hunk 1]
```diff
@@ -85,6 +85,12 @@
                         ],
                         "displayName": "Ai SkyDome Light in Hydra for Maya."
                     },
+					"MayaHydraAiAreaLightAdapter": {
```
Comment (lilike-adsk) [inline lib/mayaHydra/hydraExtensions/plugInfo.json]: remove the extra space?

Comment (lanierd-adsk) [reply lib/mayaHydra/hydraExtensions/plugInfo.json]: I have tried right now, but there is no extra space it seems.

Comment (lanierd-adsk) [reply lib/mayaHydra/hydraExtensions/plugInfo.json]: image

Comment (lilike-adsk) [reply lib/mayaHydra/hydraExtensions/plugInfo.json]: I think it's tab, you can enable option in VS "Eidt"->"Advanced"->"View White Space"

tab

Comment (lanierd-adsk) [reply lib/mayaHydra/hydraExtensions/plugInfo.json]: Seems ok now. Thanks.

Comment (lilike-adsk) [reply lib/mayaHydra/hydraExtensions/plugInfo.json]: BTW, I ususally force "Tab" as "Space" in VS when you're using VS to Edit file, as "Tab" is system dependent.
set-tab-vs

Comment (lanierd-adsk) [reply lib/mayaHydra/hydraExtensions/plugInfo.json]: Should be good now , thanks

**Review by lilike-adsk — APPROVED** [review]

**Pull request merged**